test/second_q/operators/test_sparse_label_op.py

```diff
@@ -143,6 +143,12 @@ def test_add(self):
 
             self.assertEqual(test_op, target_op)
 
+        with self.subTest("parameter + parameter"):
+            test_op = DummySparseLabelOp({"+_0 -_1": a}) + DummySparseLabelOp({"+_1 -_0": a})
+            target_op = DummySparseLabelOp({"+_0 -_1": a, "+_1 -_0": a})
+
+            self.assertEqual(test_op, target_op)
+
```

Comment:
Please also add a unittest which asserts that the proposed change works correctly when two different parameters are present (which would break without the `allow_unknown_parameters` or my proposed workaround).

Reply:
I added such a test to the test of `assign_parameters`. I realized that this test of addition was unrelated and not broken so I removed it.